scripts/deploy-wc.sh

```diff
@@ -50,9 +50,9 @@ declare -a helmfile_opt_flags
 [[ -n "$INTERACTIVE" ]] && helmfile_opt_flags+=("$INTERACTIVE")
```

Comment:
We don't use this anymore it seems.
Can you remove it (and everything that goes with it)?
```suggestion
```

Reply:
`"${*}"` is used when you want to pass all arguments as a single string. However, `${*}` (without quotes) is generally a bad idea as that would both word-split and glob expand etc. I believe it's the same for `${@}`.

On the other hand `"${@}"` will pass the arguments as an array, so that means that `$1` and `$2` stays the same between functions even if `$1` is `a*` and $2 is `a b c`. This is probably what you want in most cases.